### ISSUE TITLE
'map all' values

### ISSUE THREAD

**Issue by rlzijdeman** [RinkeHoekstra/qber#15]
Hi, I'm trying to map a column with year values (e.g. 1812, 1978) to 

http://purl.org/linked-data/sdmx/2009/code#timeFormat-602 a skos:Concept, sdmx:Concept, sdmx-code:TimeFormat;
    skos:topConceptOf sdmx-code:timeFormat;
    skos:inScheme     sdmx-code:timeFormat;
    skos:prefLabel "CCYY"@en ;
    skos:notation "602" ;
    skos:definition """(for annual series)
year"""@en ; 

The problem is that for each year I need to 'select code' and then browse the dropdown, and select CCYY, for each row in the dataset. Could we not have checkboxes for each of the values and additionally a 'revert selection' (so you can select all, or deselect all?)

**Comment by RinkeHoekstra** [comment]
Yes, this should be part of a larger effort to improve the UI for annotating datasets (including mapping-suggestions based on string matching)

**Comment by rlzijdeman** [comment]
By coincidence, Auke suggested today to have an optional regex feature to map values...

On 23 Jun 2015, at 14:45, Rinke Hoekstra <notifications@github.com<mailto:notifications@github.com>> wrote:

Yes, this should be part of a larger effort to improve the UI for annotating datasets (including mapping-suggestions based on string matching)

**Comment by RinkeHoekstra** [comment]
Oh... you mean the regex function I removed from the earlier prototype? ;-)

**Comment by rijpma** [comment]
:) Yes, though here the idea is not to split variables, but to group in value lists (e.g. combine 'bakker' and 'backer' with regex).

**Comment by RinkeHoekstra** [comment]
Possible duplicate with #22
